docs/hyperopt.md

```diff
@@ -200,7 +199,9 @@ class SuperDuperHyperOptLoss(IHyperOptLoss):
 
 Currently, the arguments are:
 
-* `results`: DataFrame containing the result
+* `results`: DataFrame containing the result  
+    The following columns are available in results (corresponds to the output of backtesting):  
```

Comment:
"corresponds to the output of backtesting" -- this is a bit incorrect since output of backtesting contains results aggregated by pair name while this list contains details on the per-trade basis with no aggregation.

Reply:
Well it's the "output-file" of backtesting (but agreed, could be clearer).